lldb/source/Utility/FileSpecList.cpp

```diff
@@ -108,52 +113,85 @@ size_t SupportFileList::FindFileIndex(size_t start_idx,
                          });
 }
 
-size_t SupportFileList::FindCompatibleIndex(size_t start_idx,
-                                            const FileSpec &file_spec) const {
-  const size_t num_files = m_files.size();
-  if (start_idx >= num_files)
-    return UINT32_MAX;
+enum IsCompatibleResult {
```

Comment:
noMatch and onlyFileMatch are pretty clear, but it would be nice to say exactly what you mean by compatible here.  For instance, I was a little surprised that exactly equal is kCompatible?